inspirehep/modules/records/receivers.py

```diff
@@ -366,6 +367,33 @@ def populate_earliest_date(sender, json, *args, **kwargs):
             json['earliest_date'] = result
 
 
+def populate_experiment_suggest(sender, json, *args, **kwargs):
+    """Populates experiment_suggest field of experiment records."""
+
+    # FIXME: Use a dedicated method when #1355 will be resolved.
+    if 'experiments.json' not in json.get('$schema'):
+        return
+
+    name_variants = force_list(json.get('name_variants', []))
+    legacy_name = json.get('legacy_name', '')
+
+    input_values = []
+    input_values.append(legacy_name)
+    input_values.extend(name_variants)
```

Comment:
for this part, maybe @michamos can tell if we should put more fields from schema that would be useful for autocompletion.

Reply:
you can also add `accelerator.value`, `collaboration.value`, `experiment.value`, `experiment.short_name`, `institutions.value`, `long_name`.